server/cli/convert_to_cxg.py

```diff
@@ -0,0 +1,126 @@
+         "usually with the generated CXG file.",
+)
+@click.argument(
+    "input-file",
+    nargs=1,
+    help="Path to the H5AD input file to be converted.",
+    type=click.Path(exists=True, dir_okay=False),
+)
+@click.option(
+    "-o",
+    "--output-dir",
+    help="Name of the output CXG directory. If not provided, will default to be the input filename with a "
+         "CXG extension.",
+)
+@click.option(
+    "-b",
+    "--backed",
+    help="When true, loads the H5AD in file backed mode. This will cause the conversion to be slower, "
+         "but will use less memory.",
+    default=False,
+    show_default=True,
+    is_flag=True,
+)
+@click.option(
+    "-t",
+    "--title",
+    help="Human readable dataset title that will be included as metadata about the CXG file. If omitted, "
+         "the dataset title will be the filename.",
+)
+@click.option(
+    "-a",
+    "--about",
+    help="A fully qualified URL that provides more information about the dataset and will be included as "
+         "metadata about the CXG file.",
+)
+@click.option(
+    "-s",
+    "--sparse-threshold",
+    help="If the dataset's percent of non-zero values falls belows the specified threshold, then the X "
+         "array of the dataset will be sparse. Since the default value is 0.0, the default will be to "
+         "convert to dense array.",
+    default=0.0,
+    show_default=True,
+)
+@click.option("--obs-names", help="The name of the observation annotations.")  # TODO expand this
+@click.option("--var-names", help="The name of the variables annotations.")  # TODO expand this
+@click.option(
+    "--disable-custom-colors",
+    help="When set, conversion process will not extract scanpy-compatible category colors from the H5AD " "file.",
+    default=False,
+    show_default=True,
+    is_flag=True,
+)
+@click.option(
+    "--disable-corpora-schema",
+    "When set, conversion process will neither extract nor store Corpora schema information. See "
+    "https://github.com/chanzuckerberg/corpora-data-portal/blob/main/backend/schema/corpora_schema.md for "
+    "more information.",
+    default=False,
+    show_default=True,
+    is_flag=True,
+)
+@click.option(
+    "--overwrite",
+    help="When set to true, will overwrite the output file if the output file already exists.",
+    default=False,
+    show_default=True,
+    is_flag=True,
+)
+@click.option("-v", "--verbose", count=True)
+@click.help_option("--help", "-h", help="Show this message and exit.")
+def convert_to_cxg(
+        input_file,
+        output_directory,
+        backed,
+        title,
+        about,
+        sparse_threshold,
+        obs_names,
+        var_names,
+        disable_custom_colors,
+        disable_corpora_schema,
+        should_overwrite,
+):
+    """
+    Convert a dataset file into CXG.
+    """
+
+    h5ad_data_file = H5ADDataFile(input_file, backed, title, about, obs_names, var_names, not disable_corpora_schema)
```

Comment:
`not disable_corpora_schema` seems a little odd, maybe you could add in the arg names to make it clear why you are reversing the boolean?

Reply:
Great suggestion! Done.